### PR TITLE
Sample Templatized Policies

### PR DESCRIPTION
**Pull request by ckandag**
Signed-off-by: Chaitanya Kandagatla <ckandaga@redhat.com>

### CODE REVIEW

File: community/CM-Configuration-Management/policy-sriovnetwork-templatized.yaml [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+apiVersion: policy.open-cluster-management.io/v1
+kind: Policy
+metadata:
+    name: policy-site-nw-templatized
+    annotations:
+        policy.open-cluster-management.io/categories: CM Configuration Management
+        policy.open-cluster-management.io/controls: CM-2 Baseline Configuration
+        policy.open-cluster-management.io/standards: NIST SP 800-53
+spec:
+    remediationAction: enforce
```
Comment (gparvin) [inline community/CM-Configuration-Management/policy-sriovnetwork-templatized.yaml]: We want policies to be inform by default.

File: community/CM-Configuration-Management/policy-sriovnetwork-templatized.yaml [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+apiVersion: policy.open-cluster-management.io/v1
+kind: Policy
+metadata:
+  name: policy-site-nw-templatized-config
+  annotations:
+    policy.open-cluster-management.io/standards: NIST 800-53
+    policy.open-cluster-management.io/categories: CM Configuration Management
+    policy.open-cluster-management.io/controls: CM-2 Baseline Configuration
+spec:
+  remediationAction: enforce
```
Comment (gparvin) [inline community/CM-Configuration-Management/policy-sriovnetwork-templatized.yaml]: We want policies to be inform by default

**Review by gparvin — COMMENTED** [review]
Can you make the policies inform by default.  Thanks for this great contribution!

**Comment by ckandag** [comment]
@gparvin  : I updated the remediation, could you pls take a look again!

**Review by gparvin — APPROVED** [review]
Looks good!

**Pull request merged**